pages/common/pg_dump.md

```diff
@@ -17,3 +17,11 @@
 - Dump a database into a custom-format archive file:
 
 `pg_dump -Fc {{db_name}} > {{output_file.dump}}`
+
+- Dump only database data into a SQL-script file:
```

Comment:
Perhaps this will read better?

```suggestion
- Dump only database data into an SQL-script file:
```

Reply:
Hi @sbrl 

I think it's **a** because the next word starts with consonant.

Reply:
True, but there are exceptions to the rule. In this case, it sounds better if we use `an` instead of `a`. It's because SQL is an abbreviation, I believe.

Reply:
English is not my main language :relaxed: 
So I know just the basic rules, I'll take the suggestions

Reply:
To me, `an` doesn't sound right. :p

But that's cause I pronounce it Sequel, rather than S-Q-L. `An` is correct though. :+1: